README.md

```diff
@@ -1,65 +1,21 @@
 Bowling Challenge in Ruby
 =================
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday week
+This is a partial solution as I didn't have time in the weekend to focus on the challenge.
 
-## The Task
+Apart from Makers' readme, I used different resource to get familiar with the bowling scoring system and its logic:
+* [How to Keep Score in Bowling](https://www.youtube.com/watch?v=-4wNjb9F1mo)
+* [The Rules of Ten Pin (10 Pin) Bowling - EXPLAINED!](https://www.youtube.com/watch?v=E2d8PizMe-8)
+* [Scoring Bowling](https://www.youtube.com/watch?v=aBe71sD8o8c)
 
-**THIS IS NOT A BOWLING GAME, IT IS A BOWLING SCORECARD PROGRAM. DO NOT GENERATE RANDOM ROLLS. THE USER INPUTS THE ROLLS.**
+I created a diagram of an open spare:
+![Open Spare](images/open-spare.png)
```

Comment:
All good planning and good diagram - really helps to get an idea of how the game works.